plugins/AMFReader/plugin.json

```diff
@@ -3,5 +3,5 @@
     "author": "fieldOfView",
     "version": "1.0.0",
     "description": "Provides support for reading AMF files.",
-    "api": "6.0.0"
+    "api": "7.0.0"
```

Comment:
To keep it consistent with the others, I'd change this to `7.0`

Reply:
It must be a semver, so it's better to have it like 7.0.0